src/rrules/blas.jl

```diff
@@ -24,6 +24,8 @@ end
 const MatrixOrView{T} = Union{Matrix{T},SubArray{T,2,<:Array{T}}}
 const VecOrView{T} = Union{Vector{T},SubArray{T,1,<:Array{T}}}
 const BlasRealFloat = Union{Float32,Float64}
+const BlasComplexFloat = Union{ComplexF32,ComplexF64}
+const BlasRealOrComplexFloat = Union{BlasRealFloat,BlasComplexFloat}
```

Comment:
I think this is equal to `BLAS.BlasFloat`, so maybe just import that [here](https://github.com/compintell/Mooncake.jl/blob/6c99c65600345800ba07829683cd469494eeed6c/src/Mooncake.jl#L46)?

Reply:
```suggestion
```